stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -90,10 +224,13 @@ public protocol DistributedActor: AnyActor, Identifiable, Hashable
 
 @available(SwiftStdlib 5.7, *)
 extension DistributedActor {
+
+  /// A distributed actor's hash and equality is implemented by directly delegating to it's ``id``.
```

Comment:
Maybe add - for more details see `# Hashable and Identifiable conformance` section?